sec.club/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import groupme from "../../assets/groupme.svg"
+import "./Footer.scss"
+
+export default function Footer(){
+    return (
+        <div class="footer">
+            <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/all.css"/>
+            <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.1/css/v4-shims.css"/>
```

Comment:
These should go in the HTML template, but regardless I think we should just find and use SVG icons. We don't need the user to download an entire font with a bunch of extra icons that will never be used.